services/gateway/apis/http-version2/methods/transactions.js

```diff
@@ -36,6 +36,7 @@ module.exports = {
 		data: { optional: true, type: 'string' },
 		limit: { optional: true, type: 'number', min: 1, max: 100 },
 		offset: { optional: true, type: 'number', min: 0 },
+		includePending: { optional: true, type: 'boolean', min: 1 },
```

Comment:
Please consider updating it to and test the same:
```suggestion
		includePending: { optional: true, type: 'boolean', convert: true, default: false },
```

Reply:
@sameersubudhi What does the `convert: true,` do here?

Reply:
`convert: true` is a sanitizer and will translate the input params to `true` or `false`.
Refer: https://github.com/icebob/fastest-validator#boolean

Reply:
I applied the suggestions 👍